.github/workflows/release-pkg.yml

```diff
@@ -79,15 +79,22 @@ jobs:
           token: ${{ secrets.PAT_GHOST }}
       - name: Update lightning version
         run: |
-          import json, os
-          fname = "versions.json"
-          with open(fname, encoding="utf-8") as fopen:
+          import json, os, re
+          tag = os.getenv('TAG')
+          fname_json = "versions.json"
+          with open(fname_json, encoding="utf-8") as fopen:
               vers = json.load(fopen)
-          vers["lightning_version"] = os.getenv('TAG')
-          with open(fname, "w", encoding="utf-8") as fopen:
+          vers["lightning_version"] = tag
```

Comment:
here you also need to bump the conda snapshot version from current X.Y.Z to X.Y.(Z+1)

Reply:
```suggestion
          vers["lightning_version"] = tag
          conda_ = vers["conda_snapshot"].split(".")
          conda_[-1] = str(int(conda_[-1]) + 1)
          vers["conda_snapshot"] = ".".join(conda_)
```